lib/main.dart

```diff
@@ -105,6 +109,16 @@ class _RandomWordsState extends State<RandomWords> {
         title: const Text('Startup Name Generator'),
         actions: [
           IconButton(onPressed: _pushSaved, icon: Icon(Icons.list)),
+          IconButton(
+              onPressed: () {
+                setState(() {
+                  _rowColor = !_rowColor; // This change Container color
```

Comment:
This comment is ambiguous. Container sounds too generic. `_rowColor `actually changes icon color.

Reply:
Forgot to remove, thanks for noticing)